Code/cmake/Modules/RDKitUtils.cmake

```diff
@@ -9,6 +9,10 @@ set(RDKit_RELEASENAME "${RDKit_Year}.${RDKit_Month}")
 if (RDKit_Revision)
   set(RDKit_RELEASENAME "${RDKit_RELEASENAME}.${RDKit_Revision}")
   set(RDKit_VERSION "${RDKit_VERSION}.${RDKit_Revision}")
+  if (RDKit_RevisionModifier)
```

Comment:
Do you really need this "if statement"? Won't this always be defined, either as empty string or having a value? (Same goes for the outer "if statement" about RDKit_Revision)

Reply:
good point. that's been revised